pkg/api/admin/openshiftcluster.go

```diff
@@ -295,3 +296,7 @@ type SystemData struct {
 	LastModifiedByType CreatedByType `json:"lastModifiedByType,omitempty"`
 	LastModifiedAt     *time.Time    `json:"lastModifiedAt,omitempty"`
 }
+
+type HiveProfile struct {
```

Comment:
this can be removed by using the doc uuid

Reply:
Resolving thread in favour of https://github.com/Azure/ARO-RP/pull/2145#discussion_r913319980